system/lib/libc-wasm.symbols

```diff
@@ -1,12 +1,31 @@
 # Auto-generated by tools/update_symbols.py.  DO NOT EDIT.
+         U __addtf3
```

Comment:
were the changes here made by that automatic tool? odd they are different then, as I'm not sure why this PR would change them.

cc @sbc100 who wrote that tool.

Reply:
Removed the changes made by the tool.